portal-ui/src/main/java/com/agiletec/aps/system/services/controller/control/RequestValidator.java

```diff
@@ -198,20 +198,23 @@ protected String getNotFoundPageCode() {
     }
 
     protected ILangManager getLangManager() {
-        return _langManager;
+        return langManager;
     }
 
     public void setLangManager(ILangManager langManager) {
-        this._langManager = langManager;
+        this.langManager = langManager;
     }
 
-    private ILangManager _langManager;
+    private transient ILangManager langManager;
 
+    /**
+     * @deprecated pattern for old suffix ".wp"
```

Comment:
you should add something like "you should use the new `pattern` variable", when you deprecate you should always explain the new solution, not only deprecation reason

Reply:
This pattern in maintained to ensure compatibility with links used in older installations. Explained in javadoc